Dockerfile

```diff
@@ -1,7 +1,7 @@
 ### STAGE 1: Build ###
 
 # We label our stage as 'builder'
-FROM maven:alpine as builder
+FROM maven:3.6.0-jdk-12-alpine as builder
 
```

Comment:
Is it better to generate a docker image with the same version or with the next one?

Reply:
I had the same question when I saw the upgrade to openJDK11 on the seed-java and I understood that this is the way that you want to do regarding Docker images. So, do you agree with me to keep this line?:

`FROM maven:alpine as builder`

Reply:
I put this version in seed-java because threre isn't any jdk-11-alpine. I understood that we wanted to use alpine images because are lightweight images. If not we can use a jdk-11 image